### PR TITLE
Update structured log viewer to latest

### PR DESCRIPTION
**Pull request by sebastienros**
Fix aspnetcore CI builds

`Unhandled exception. System.NotSupportedException: Unsupported log file format. Latest supported version is 16, the log file has version 17.`

This version supports 17

### CODE REVIEW

**Review by riarenas — APPROVED** [review]
This lines up with other updates we've done. So this LGTM I guess? No idea how this gets deployed, and I'm not sure it's expected for this repo to not run any builds during PRs?

**Comment by riarenas** [comment]
@garath maybe knows something?

**Comment by riarenas** [comment]
It looks like this might just get deployed straight to production after merging when the CI pipeline runs? 

We also have merged a bunch of other PRs without any PR validation, so it looks like this is just the state of things right now.

**Comment by sebastienros** [comment]
Note that I don't have the merge rights, so don't please wait on me ;)

**Comment by garath** [comment]
I agree this is inline with previous changes. Let's take this and check the state of the main build after the nightly run.

**Review by garath — APPROVED** [review]

**Comment by garath** [comment]
By request I've started an early build to get this published: [Pipelines - Run 20231027.2 (azure.com)](https://dev.azure.com/dnceng/internal/_build/results?buildId=2303032&view=results)

**Comment by riarenas** [comment]
Couldn't this wait until Monday? We shouldn't get in the habit of deploying services on a Friday

**Comment by sebastienros** [comment]
> Couldn't this wait until Monday? We shouldn't get in the habit of deploying services on a Friday

Right, it could break the SourceIndex build steps. Oh wait! 😁

No worries, there might be other things that get broken. I can't estimate the impact of these change and I don't think it's a big problem for a WE.

**Comment by garath** [comment]
I hear you, it is not a risk-free thing.

CI/CD normally runs every night, so this would have happened in a few hours anyway.

**Comment by sebastienros** [comment]
I assume nothing will get through until another change gets into arcade, right?

https://github.com/dotnet/arcade/blob/main/eng/common/templates/job/source-index-stage1.yml#L3

**Pull request merged**
